sqlglot/dialects/exasol.py

```diff
@@ -21,4 +21,28 @@ class Generator(generator.Generator):
         TYPE_MAPPING = {
             **generator.Generator.TYPE_MAPPING,
             **STRING_TYPE_MAPPING,
+            # https://docs.exasol.com/db/latest/sql_references/data_types/datatypealiases.htm
+            exp.DataType.Type.TINYINT: "SMALLINT",
+            exp.DataType.Type.INT: "INTEGER",
```

Comment:
This isn't necessary, `INT` and `INTEGER` are aliases to the same thing.

Reply:
You're right I would remove it  @georgesittas